Library/Formula/cccc.rb

```diff
@@ -0,0 +1,58 @@
+class Cccc < Formula
+  desc "C and C++ Code Counter"
+  homepage "http://sourceforge.net/projects/cccc/"
+  url "https://downloads.sourceforge.net/project/cccc/cccc/3.1.4/cccc-3.1.4.tar.gz"
+  sha256 "27b3eca9a83a66799332363a80cc4bcd6db9869eddbda1a59a25cdace3ff4dbf"
+
+  # reported to upstream: https://sourceforge.net/p/cccc/bugs/55/
+  patch :DATA
+
+  def install
+    # "make cccc" will fail if parallelized.
```

Comment:
Aye, ideally we can ask upstream about root. If they turn around and tell us they're not changing it that's their choice, but ideally in these situations `su`/`sudo` invocation should be left to user choice rather than hardcoded like this.